examples/evaluate_hps.py

```diff
@@ -0,0 +1,56 @@
+# Temporary script for Development Purposes
+from tempfile import TemporaryDirectory
+
+from embeddings.hyperparameter_search.configspace import (
+    SequenceLabelingConfigSpace,
+    TextClassificationConfigSpace,
+)
+from embeddings.pipeline.flair_hps_pipeline import (
+    OptimizedFlairClassificationPipeline,
+    OptimizedFlairSequenceLabelingPipeline,
+)
+from embeddings.pipeline.hugging_face_classification import HuggingFaceClassificationPipeline
+from embeddings.pipeline.hugging_face_sequence_labeling import HuggingFaceSequenceLabelingPipeline
+
+
+def main() -> None:
```

Comment:
align with previous examples (typer etc.)

Reply:
Moved to readme.md